src/test/java/api/requests/scenarios/LoanDueDatesAfterRecallTests.java

```diff
@@ -525,6 +526,400 @@ public void changedDueDateAfterRecallingAnItemShouldRespectTenantTimezone()
       storedLoan.getString("dueDate"), isEquivalentTo(expectedDueDate));
   }
 
+  @Test
+  public void pagedItemRecalledThenLoanedAndNextRecallDoesNotChangeDueDate()
+      throws InterruptedException,
+      ExecutionException,
+      TimeoutException,
+      MalformedURLException {
+    final IndividualResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource requestServicePoint = servicePointsFixture.cd1();
+    final IndividualResource jessica = usersFixture.jessica();
+    final IndividualResource charlotte = usersFixture.charlotte();
+    final IndividualResource james = usersFixture.james();
+
+    final LoanPolicyBuilder canCirculateRollingPolicy = new LoanPolicyBuilder()
+        .withName("Can Circulate Rolling With Recalls")
+        .withDescription("Can circulate item With Recalls")
+        .rolling(Period.weeks(3))
+        .unlimitedRenewals()
+        .renewFromSystemDate()
+        .withRecallsMinimumGuaranteedLoanPeriod(Period.weeks(2))
+        .withRecallsRecallReturnInterval(Period.weeks(1));
+
+    final IndividualResource loanPolicy = loanPoliciesFixture.create(canCirculateRollingPolicy);
+
+    useLoanPolicyAsFallback(loanPolicy.getId(),
+        requestPoliciesFixture.allowAllRequestPolicy().getId(),
+        noticePoliciesFixture.activeNotice().getId());
```

Comment:
Do notices need to be active for this test?

Reply:
That's a good question. I know very little about notices or how they work in the tests. Should I use `noticePoliciesFixture.inactiveNotice().getId()`? This code was added when CIRC-203 was implemented.

Reply:
Yes please. I've been starting to do that in places where polices are set and the test doesn't involve notices.

I've been wondering if there should be something stateful that knows about the current policies and rules, so can instead only change the policy the test cares about.

Reply:
Could the code to set up loan policy be extracted out for reuse and to de-clutter the tests, if it's the same for every test?

Reply:
The loan policy itself is not the same in every test, though there is commonality between several of them. Maybe we can do this as follow on work? Perhaps some of this could be moved to the APITests base class so tests in other domains could use policies that have minimum guaranteed durations and recall intervals set for tests that involve recalls.

Reply:
Sure, a follow on work is fine. This is not a big deal.  You're right, the loan policy itself is not the same. By that I meant the useLoanPolicyAsFallback() call.

Reply:
The configuration of circulation rules in the tests needs significant work.

The current approach of using the API test base methods was a compromise that allowed many of the tests to have a sensible default and the ability to change it relatively easily.

I'd prefer some form of stateful actor that looked after the rules.